template.json

```diff
@@ -10,41 +10,77 @@
             "background": {
                 "biography": ""
             },
-            "resources": {
-                "hitPoints": {
-                    "min": 0,
-                    "value": 10,
-                    "max": 40
-                },
-                "pietas": {
-                    "min": 0,
-                    "value": 10,
-                    "max": 50
+            "common": {
+                "attributes":
+                {
+                    "hitPoints": {
+                        "min": 0,
+                        "value": 10,
+                        "max": 40
+                    },
+                    "pietas": {
+                        "min": 0,
+                        "value": 10,
+                        "max": 50
+                    }
                 }
             }
         },
         "custos": {
             "templates": [
                 "background",
-                "resources"
+                "common"
             ],
+            "peritiae":
+            {
+                "deBello":
+                {
+                    "value": 14,
+                    "defaultRoll": "d10+d4"
+                },
+                "deNatura":
+                {
+                    "value": 17,
+                    "defaultRoll": "d12+d5"
+                },
+                "deSocietate":
+                {
+                    "value": 6,
+                    "defaultRoll": "d6"
+                },
+                "deMagia":
+                {
+                    "value": 8,
+                    "defaultRoll": "d8"
+                },
+                "deScientia":
+                {
+                    "value": 6,
+                    "defaultRoll": "d6"
+                },
+                "deCorpore":
+                {
+                    "value": 14,
+                    "defaultRoll": "d10+d4"
+                }
+            },
```

Comment:
Pretty sure that peritiae have minimum values (maybe 2? I don't really remember). I don't see the use of defining a default roll at that point because it will never be used, since users do pick their rolls every time they roll the peritia. Do you have an example of the use of such an attribute?

Reply:
Good catch for the minimum, but as this is a general rule, maybe we can put it in the system directly, like upstairs in the object. For the default roll, I think its a good idea since I observed that players use to roll the same dices often, so it will allow to use a default roll with a button and it will help speed up things. And when players will want to change it they could aside.